src/lib/modules/tests/test.js

```diff
@@ -32,7 +32,7 @@ class Test {
       return callback();
     }
     if (!this.ipc.connected) {
-      this.engine.logger.error("Could not connect to Embark's IPC. Is embark running?");
+      this.logger.error("Could not connect to Embark's IPC. Is embark running?");
```

Comment:
This was erroring with `engine` being `null`:
1. embark run in the test app
2. In another term, embark test —node embark (in the test app)
3. Error attempted to be logged, but `engine` is `null`